### PR TITLE
Export as graph PNG

### PR DESCRIPTION
**Pull request by ryanwmao**
Example in Graph.png

### CODE REVIEW

File: GUI/widgets/plot_buttons.py [hunk 1]
```diff
@@ -1,12 +1,21 @@
...
         super(PlotButtons, self).__init__(None)
         self.plot = plot
         self.plot_reset = False
+        # self.exporter = pyqtgraph.exporters.ImageExporter(
+        #     self.plot.scene())
+        # self.exporter.parameters()['width'] = 1372
+        # self.exporter.parameters()['height'] = 639
+        # self.exporter.parameters()['width'], \
+        #     self.exporter.parameters()['height'] = \
+        #     self.plot.frameGeometry().width(), self.plot.frameGeometry().height()
```
Comment (roninsharma25) [inline GUI/widgets/plot_buttons.py:18]: Can you delete these lines if they aren't used?

File: GUI/widgets/plot_buttons.py [hunk 2]
```diff
@@ -78,20 +96,24 @@ def initUI(self):
...
     def setPlotResetFlag(self, flag):
         self.plot_reset = flag
-    
+
     def getPlotResetFlag(self):
         return self.plot_reset
 
+    def exportGraph(self):
+        self.plot.getPlotItem().writeImage("Graph.png")
+        # self.exporter.export('Graph.png')
```
Comment (roninsharma25) [inline GUI/widgets/plot_buttons.py:115]: This can be deleted as well

File: GUI/widgets/plot_buttons.py [hunk 3]
```diff
@@ -78,20 +96,24 @@ def initUI(self):
...
+
     def setPlotResetFlag(self, flag):
         self.plot_reset = flag
-    
+
     def getPlotResetFlag(self):
         return self.plot_reset
 
+    def exportGraph(self):
+        self.plot.getPlotItem().writeImage("Graph.png")
```
Comment (roninsharma25) [inline GUI/widgets/plot_buttons.py:114]: Do we want the filename to be in the constants file?

**Review by roninsharma25 — CHANGES_REQUESTED** [review]
Could you also add something that notifies the user that the graph has been downloaded? Currently, nothing happens so a user might not know what happens. One option is to have a popup similar to how the help button currently operates. Since this feature wasn't part of the initial task description, feel free to create a new task for the feature.

**Comment by ryanwmao** [comment]
Created new task for button, i think it applies to a couple other buttons on GUI as well (#271)

**Review by KelvinLWang — APPROVED** [review]

**Review by roninsharma25 — APPROVED** [review]

**Pull request merged**